JeongJaeHeon/db/migrations/20221123061126_create_users_table.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    user_name VARCHAR(100) NOT NULL,
+    user_email VARCHAR(200) NOT NULL,
+    user_password VARCHAR(1000) NOT NULL,
+    cover_image VARCHAR(1000) NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
```

Comment:
재헌님!
실제 데이터베이스를 다루면서 처음 구현해보는 회원가입 API임에도 너무 잘 해주셨습니다~!

- column 네이밍 관련해서 피드백을 드리고자 하는데, users 테이블에서 각각의 컬럼이 가리키는 의미는 이미 user라는 의미가 내포되어 있기 떄문에 `user_***` 라는 컬럼 네이밍을 사용하지 않아도 될 것 같습니다!

- 또한 유저의 데이터는 중복이 있어서는 안될 데이터라고 판단됩니다!
`email` 컬럼에 unique 속성을 지정해주세요!

```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name VARCHAR(100) NOT NULL,
    email VARCHAR(200) NOT NULL,
    password VARCHAR(1000) NOT NULL,
    cover_image VARCHAR(1000) NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE (email)
);
```